lessons/misc/ml-intro/kmeans.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: Unsupervised Learning
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+rm(list = ls())
+```
+
+## GitHub Documents
+
+This is an R Markdown format used for publishing markdown documents to GitHub. When you click the **Knit** button all R code chunks are run and a markdown file (.md) suitable for publishing to GitHub is generated.
+
+## Including Code
+
+You can include R code in the document as follows:
+
+```{r iris}
+library(datasets)
+head(iris)
+```
+
+## Including Plots
+
+You can also embed plots, for example:
+
+```{r pressure, echo=FALSE}
+first<-plot(iris)
+
+library(ggplot2)
+second<-ggplot(iris,aes(Petal.Width,Petal.Length))+geom_point()
+```
+
+We can do the same graph putting color to different  Species of flower
+
+```{r}
+ggplot(iris, aes(Petal.Length, Petal.Width, color = Species)) + geom_point()
+```
+```
+
+```
+```{r}
+set.seed(20)
+irisCluster <- kmeans(iris[, 3:4], 3, nstart = 20)
```

Comment:
You'll need to describe `kmeans` in text above the code chunk or something to explain what it does.